warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/define/EdgeDataBundle.java

```diff
@@ -97,6 +99,17 @@ public EdgeDataBundle(EdgeDefinition edgeDef, NormalizedContentInterface ifaceSo
         this.initMarkings(getSource().getMarkings(), getSink().getMarkings());
```

Comment:
The constructor ` public EdgeDataBundle(EdgeDefinition edgeDef...` can be deleted now that it is unused.  Its code now seems to exist in the data type handler's createEdge.

Reply:
I put this and a few other suggested removals in https://github.com/NationalSecurityAgency/datawave/pull/2269/files